src/DSharp.Mscorlib/Activator.cs

```diff
@@ -11,6 +11,7 @@ public static class Activator
     {
         public extern static object CreateInstance(Type type);
 
+        [ScriptIgnoreGenericArguments]
         public extern static T CreateInstance<T>();
```

Comment:
Maybe we can make the extern automatically be ScriptIgnoreGenericArguments?

Reply:
or have an [assembly:] level attribute we can apply to set the default behaviour?

Reply:
I'll look at doing this as an improvement instead of fixing it now as it could take a bit of time to implement :)